_includes/settings-table.html

```diff
@@ -50,10 +50,12 @@
           {{ setting.default }}
         </span>
       </td>
-      <td data-title="{{ col4Title }}" class="{{ col4Width }} display-inline-flex flex-align-start">
-        <span>
-          {% include get-token-url.html token=setting.type %}
-        </span>
+      <td data-title="{{ col4Title }}" class="{{ col4Width }} display-inline-flex flex-align-start flex-wrap">
+        {% for type in setting.type %}
+          <span class="margin-bottom-05">
+            {% include get-token-url.html token=type %}
+          </span>
+        {% endfor %}
```

Comment:
This allows for multiple token types to be declared. Useful for tokens like `$theme-collection-header-typeset` that need to declare three different tokens in one setting.